frontend/src/js/controllers/ourteamCtrl.js

```diff
@@ -21,28 +21,32 @@
             onSuccess: function(response) {
                 var status = response.status;
                 var results = response.data;
-                if ((status == 200) && results.length) {
-                    var coreTeamList = [];
-                    var contributingTeamList = [];
-                    for (var i = 0; i < results.length; i++) {
-                        if (results[i].team_type === "Core Team") {
-                            vm.coreTeamType = results[i].team_type;
-                            vm.coreTeamList = coreTeamList.push(results[i]);
-
-                        } else if (results[i].team_type === "Contributor") {
-                            vm.contributingTeamType = results[i].team_type;
-                            vm.contributingTeamList = contributingTeamList.push(results[i]);
+                if (status == 200) {
+                    if (results.length !== 0) {
+                        var coreTeamList = [];
+                        var contributingTeamList = [];
+                        for (var i = 0; i < results.length; i++) {
+                            if (results[i].team_type === "Core Team") {
+                                vm.coreTeamType = results[i].team_type;
+                                vm.coreTeamList = coreTeamList.push(results[i]);
+
+                            } else if (results[i].team_type === "Contributor") {
+                                vm.contributingTeamType = results[i].team_type;
+                                vm.contributingTeamList = contributingTeamList.push(results[i]);
+                            }
+                            vm.coreTeamDetails = coreTeamList;
+                            vm.contributingTeamDetails = contributingTeamList;
                         }
-                        vm.coreTeamDetails = coreTeamList;
-                        vm.contributingTeamDetails = contributingTeamList;
+                    } else {
+                        vm.noTeamDisplay = "Team will be updated very soon !";
                     }
                 }
-                else {
-                    vm.noTeamDisplay = "Team will be updated very soon !"
-                }
+            },
+            onError: function(response) {
+                vm.noTeamDisplay = "Team will be updated very soon !";
```

Comment:
Remove this line @RishabhJain2018 . This will not come in error status code. Just leave this block empty for now.